config/local_info.xml

```diff
@@ -179,6 +179,37 @@
     -->
     <restrict_personal_data>false</restrict_personal_data>
 
+    <!-- identity_providers
+      Shib/CheckIn Token reads config file
+      for needed AAI entitlements.
+      This is to allow Admins so that they will be empowered
+      to make quicker changes to AAI references.
+    -->
+    <identity_providers>
+      <provider>
+        <idp>aai.egi.eu/auth/realms/egi</idp>
+        <name>EGI Proxy</name>
+        <authentication_realms>
+          <shib_realm_name>EGI Proxy IdP</shib_realm_name>
+        </authentication_realms>
```

Comment:
This can just be `authentication_realm`

```suggestion
        <authentication_realm>
            EGI Proxy IdP
        </authentication_realm>
```

Reply:
Updated in 01f3cd379b62c4b3119a25d09f7cef0d852b9904